scenes/game/game.gd

```diff
@@ -1954,7 +1972,7 @@ func _on_gauge_for_effect(event):
 
 func _on_damage(event):
 	var player = event['event_player']
-	var life = game_wrapper.get_player_life(player)
+	var life = event['extra_info']
```

Comment:
It could be nice to have the spend_life effect call this event (maybe with something to avoid playing the animation), because otherwise it seems like your health doesn't visually update until you get hit or heal